### ISSUE TITLE
TYPO3 V13 compatibility 

### ISSUE THREAD

**Issue by mschwemer** [in2code-de/powermail#943]
There will be an EXT:powermail version, which is compatible with TYPO3 core V13. Active development will be started in Q2 / 2024. It will be publicly available about 6 -9 months after the release of TYPO3 13LTS.

If you need early access to this version, please head over to https://www.in2code.de/agentur/typo3-extensions/early-access-programm/ and support our work.

**Comment by proudnerds-typo3** [comment]
Don't you think that this is relatively late? The version for TYPO3 v12 was made public earlier.

**Comment by mschwemer** [comment]
@proudnerds-typo3 No, the v12 compatible version had exact the same period between TYPO3 v12 LTS and making the powermail compatibilty version public.

**Comment by mrkbr** [comment]
I know - making money is important, but that makes using Powermail almost pointless. If that was the way most developers supported current core versions, the whole TYPO3 project would simply be dead.

**Comment by Sofatraining** [comment]
@mschwemer 
> Active development will be started in Q2 / 2024. It will be publicly available about 6 -9 months after the release of TYPO3 13LTS.

Any news? The schedule doesn't seem to be right.

**Comment by mkrappitz** [comment]
We are now 8 months since the release of  TYPO3 13 and so far there is no info about a v13 version of powermail. Has the development started? Is there a new schedule? Or is it rather adisable to migrate to form framework instead?

**Comment by Patta** [comment]
@mkrappitz @Sofatraining just read the EAP website mentioned above: https://www.in2code.de/agentur/typo3-extensions/early-access-programm/
> available in TER from July 2025
